packages/synapse-react-client/src/components/CardContainer/CardContainer.integration.test.tsx

```diff
@@ -129,142 +91,14 @@ describe('CardContainer tests', () => {
     await screen.findByText(title)
   })
 
-  it('ViewMore button shows full pages on each click when initialLimit is not set', async () => {
```

Comment:
`CardContainer` no longer handles pagination, so all of these pagination tests were removed. New tests were created for the new components/hooks that will handle this logic.